src/app/Fake.Sql.SqlServer/Fake.Sql.SqlServer.fsproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I think this is why the api docs are missing, we need `<TargetFrameworks>netstandard2.0;net46</TargetFrameworks>` (ie one target for the full framework) in order to generate docs. This is because FSharp.Formatting is not yet compatible with netstandard assemblies.